### PR TITLE
Improve newcomers setup

### PR DESCRIPTION
**Pull request by zatteo**
4 improvement propositions from the setup from @Merkur39 

Tell me what you think :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +40,16 @@ More information about Android variant [here](https://github.com/cozy/cozy-flags
 
 ### Working with locally hosted webviews
 
+The easiest way to develop on locally hosted webviews (like any Cozy apps like Drive or Contacts) is to :
+- build your app
+- change the version number in build/manifest.webapp
```
Comment (Ldoppea) [inline README.md:45]:
```suggestion
- change the version number in the cozy-app's build/manifest.webapp
```

File: README.md [hunk 2]
```diff
@@ -69,7 +64,7 @@ You can then connect to your local stack using the following URL : `http://fooba
 #### On iOS simulator
 
 - Create a cozy instance with the \*.cozy.tools:8080 format
-- Disable the local httpServer by setting to `true` disableGetIndex in `dev.js`
+- Disable the local httpServer by setting to `true` disableGetIndex in `src/constants/dev-config.ts`
```
Comment (Ldoppea) [inline README.md:67]: Should be done on both platforms

File: README.md [hunk 3]
```diff
@@ -55,6 +40,16 @@ More information about Android variant [here](https://github.com/cozy/cozy-flags
 
 ### Working with locally hosted webviews
 
+The easiest way to develop on locally hosted webviews (like any Cozy apps like Drive or Contacts) is to :
+- build your app
+- change the version number in build/manifest.webapp
+- go to home app
```
Comment (Ldoppea) [inline README.md:46]:
```suggestion
- open the flagship app and go to home cozy-app
```

File: README.md [hunk 4]
```diff
@@ -55,6 +40,16 @@ More information about Android variant [here](https://github.com/cozy/cozy-flags
...
+- go to your app => flagship app will download the just builded version in background
+- go to home app
```
Comment (Ldoppea) [inline README.md:48]:
```suggestion
- go to home cozy-app
```

File: README.md [hunk 5]
```diff
@@ -55,6 +40,16 @@ More information about Android variant [here](https://github.com/cozy/cozy-flags
...
+- go to your app => you will be on the just builded version
```
Comment (Ldoppea) [inline README.md:49]:
```suggestion
- go to your cozy-app => you will be on the just builded version
```

File: README.md [hunk 6]
```diff
@@ -55,6 +40,16 @@ More information about Android variant [here](https://github.com/cozy/cozy-flags
 
 ### Working with locally hosted webviews
 
+The easiest way to develop on locally hosted webviews (like any Cozy apps like Drive or Contacts) is to :
+- build your app
```
Comment (Ldoppea) [inline README.md:44]:
```suggestion
- build your cozy-app
```

**Review by Ldoppea — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (zatteo) [reply README.md:67]: Actually it was already done in Android so I just update it here.

**Pull request merged**